scanners/gitleaks/parser/parser.js

```diff
@@ -0,0 +1,45 @@
+HIGH_TAGS = ['JWT', 'Artifactory', 'AWS', 'PrivateKey'];
+MEDIUM_TAGS = ['Hash', 'Facebook', 'Twitter', 'Github', 'LinkedIn', 'Slack', 'Google', 'Heroku',
+  'Mailchimp', 'Mailgun', 'Paypal', 'Picatic', 'Teams', 'Jenkins', 'Stripe', 'Square', 'Twilio'];
+LOW_TAGS = ['Generic', 'FileName', 'FileExtension'];
+
+async function parse (fileContent) {
+
+  return fileContent.map(finding => {
+
+    let severity = 'LOW';
+
+    if (containsTag(finding.tags, HIGH_TAGS)) {
+      severity = 'HIGH'
+    } else if (containsTag(finding.tags, MEDIUM_TAGS)) {
+      severity = 'MEDIUM'
+    }
+
+    return {
+      name: finding.rule,
+      description: 'The name of the rule which triggered the finding: ' + finding.rule,
+      osi_layer: 'APPLICATION',
+      severity: severity,
+      category: 'potential secret',
```

Comment:
Our other categories are all capitalized.

```suggestion
      category: 'Potential Secret',
```